packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -57,6 +58,12 @@ class ReleaseAndroidCommand extends ShorebirdCommand
             'To learn more, see: https://developer.android.com/studio/build/configure-apk-splits#configure-abi-split',
         hide: true,
         negatable: false,
+      )
+      ..addMultiOption(
+        'target-platform',
+        help: 'The target platforms for which the app is compiled.',
```

Comment:
Shouldn't this be in the singular?
```suggestion
        help: 'The target platform for which the app is compiled.',
```

Reply:
Or maybe the flag should be in plural? I just saw down in the PR a comment of it in plural

Reply:
The flag replicates what Flutter offers. It can accept multiple platforms Ex: `flutter build appbundle --target-platform=android-arm,android-arm64` works. Updated to indicate that this can be "platform" or "platforms"